packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -1422,13 +1422,17 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
 
       const { density } = this.props;
 
+      // Only get theme for explicitly set density
+      let densityTheme = {};
+      if (density != null) {
+        densityTheme = getDensityTheme(density);
```

Comment:
If you do as my other suggestion says to just return `{}` for `regular`, then you don't need this `undefined` check and can always just set the density, simplifying this.

Reply:
Would need to connect  `IrisGrid` to redux to do that or create a new context to provide the default density to `IrisGrid`. Could modify the `IrisGridThemeContext` to contain theme + density, I guess, but this seemed cleaner